app/models/grouping.rb

```diff
@@ -744,10 +744,10 @@ def get_total_test_script_marks
 
     #add the latest result from each of our test scripts
     test_script_ids.each do |test_script_id|
-      test_result = TestScriptResult.where(:test_script_id => test_script_id, :grouping_id => self.id).last
+      test_result = TestResult.where(:test_script_id => test_script_id, :grouping_id => self.id).last
```

Comment:
Line is too long. [101/80]<br>Use the new Ruby 1.9 hash syntax.<br>Redundant `self` detected.